receiver/k8sobjectsreceiver/README.md

```diff
@@ -36,9 +36,33 @@ The following is example configuration
         mode: watch
         group: events.k8s.io
         namespaces: [default]
+
+  k8sobjects/leader-election:
+    leader_election:
+      enabled: true
+      lock_name: "k8sobjects-leader-election"
+    auth_type: serviceAccount
+    objects:
+      - name: pods
+        mode: pull
+        label_selector: environment in (production),tier in (frontend)
+        field_selector: status.phase=Running
+        interval: 15m
+      - name: events
+        mode: watch
+        group: events.k8s.io
+        namespaces: [default]
 ```
 
 Brief description of configuration properties:
+- `leader_election`: running in Kubernetes with `leader election mode`, this means that multiple instances are running, 
```

Comment:
What happens if I have two instances of this component for the same collector? Is only one of them going to be elected leader? Why would I use this? Should I always use this, or only on specific situations?

Reply:
I updated the docs.